### PR TITLE
[13.0][IMP] account_invoice_overdue_reminder: changes to use cron

### PR DESCRIPTION
**Pull request by JudithAforgeFlow**
Added with_context() to avoid check warnings when a cron is used.

CC @ForgeFlow

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
Do the warnings prevent the cron to send the reminders? if so, then this LGTM

File: account_invoice_overdue_reminder/wizard/overdue_reminder_wizard.py [hunk 1]
```diff
@@ -180,7 +181,10 @@ def run(self):
                 action = orso.create(vals)
                 action_ids.append(action.id)
         if not action_ids:
-            raise UserError(_("There are no overdue reminders."))
+            if reminder_cron_execution:
+                return action
```
Comment (JordiMForgeFlow) [inline account_invoice_overdue_reminder/wizard/overdue_reminder_wizard.py]:
```suggestion
                return False
```

**Comment by MiquelRForgeFlow** [comment]
> Added with_context() to avoid check warnings

Where is it added?

**Review by JordiMForgeFlow — APPROVED** [review]
LGTM 👍🏼

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

**Review by AaronHForgeFlow — COMMENTED** [review]
/ocabot merge patch

**Pull request merged**